src/aat/java/uk/gov/hmcts/reform/coh/functional/bdd/steps/AnswerSteps.java

```diff
@@ -215,13 +215,13 @@ public void send_request(String type) throws IOException {
             } else if ("POST".equalsIgnoreCase(type)) {
                 HttpEntity<String> request = new HttpEntity<>(json, header);
```

Comment:
The only comment I would have is to update the create answer JSON (resources/json/answers) with valid answer state.

Reply:
Done